pkg/packages/install.go

```diff
@@ -173,36 +175,31 @@ type InstallOrUpdateBinaryOptions struct {
 }
 
 // ShouldInstallBinary checks if the given binary should be installed
-func ShouldInstallBinary(name string) (fileName string, download bool, err error) {
-	fileName = BinaryWithExtension(name)
-	download = false
-	pgmPath, err := exec.LookPath(fileName)
```

Comment:
this needs to happen conditionally depending on whether this is running in cluster or not. Otherwise we get the build image downloading third-party dependencies. In a follow up step we can try to improve on this.